tutorials/dataframe/df021_createTGraph.C

```diff
@@ -0,0 +1,33 @@
+/// \file
+/// \ingroup tutorial_dataframe
+/// \notebook
+/// This tutorial shows how to fill a TGraph using the Dataframe.
+///
+/// \macro_code
+///
+/// \date July 2018
+/// \author Enrico Guiraud, Danilo Piparo, Massimo Tumolo
+
+
+
+void df021_createTGraph()
+{
+   std::vector<int> source(160);
+   for (int i = 0; i < 160; ++i)
+      source[i] = i;
+
+   ROOT::RDataFrame d(160);
+   auto dd = d.DefineSlotEntry("x1",
+                               [&source](unsigned int slot, ULong64_t entry) {
```

Comment:
Why aren't we using an integer variable which we capture by reference?
```.cpp
unsigned int x_val = 0;
auto dd = d.DefineSlotEntry("x1", [&x_val]() { return x_val++; } ....
```

Reply:
It is not thread safe I think